docs/querying-data.rst

```diff
@@ -0,0 +1,153 @@
+Querying data
+=============
+
+Before you begin, read the :doc:`index` and :doc:`data-model` pages to learn about how data is stored in Kingfisher Process.
+
+Since most analysis is much easier to perform on compiled releases, we recommend working with compiled release collections to begin with.
```

Comment:
Key part of working practice introduced at the start; nice :-) I think this will assist users and future helpdesk analysts.